src/openrct2/ride/Ride.cpp

```diff
@@ -1984,7 +1984,7 @@ void Ride::UpdateAll()
     // Remove all rides if scenario editor
     if (gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR)
     {
-        if (gS6Info.editor_step <= EDITOR_STEP_INVENTIONS_LIST_SET_UP)
+        if (EnumValue(gS6Info.editor_step) <= EnumValue(EditStep::InventionsListSetUp))
```

Comment:
Can you create an issue to change the type of `gS6Info.editor_step`?

Reply:
Shouldn't this be a switch with all of the relevant steps.

Reply:
I can convert it to a switch no problem.

O have already changed the type to the enuk one, haven't I? Not sure I get you Gym

Reply:
I was thrown off by the use of `EnumValue`, but I now see why you had to use that.